packages/world/ts/getWorldAbi.ts

```diff
@@ -20,7 +24,10 @@ export async function getWorldAbi({
     fromBlock,
     toBlock,
   });
-  const worldFunctionsAbi = worldFunctions.map((func) => functionSignatureToAbiItem(func.signature));
+  const baseFunctionSelectors = (IBaseWorldAbi as Abi).filter(isAbiFunction).map(toFunctionSelector);
+  const worldFunctionsAbi = worldFunctions
+    .map((func) => functionSignatureToAbiItem(func.signature))
+    .filter((abiItem) => !baseFunctionSelectors.includes(toFunctionSelector(abiItem as AbiFunction)));
```

Comment:
Adjusted to  return `AbiFunction` but in order for that to work, have to ensure that it indeed returns function ABI item, otherwise the function throws.